gsheet_data.py

```diff
@@ -44,8 +44,8 @@ def get_raw_gsheet_data(sheet_id):
        
     return final_gsheet
 
-  except:
-    sheet_err.append('Requires access to sheet or sheet has data without headers')
+  except ValueError as e:
```

Comment:
Here the try block is too wide. You need to narrow it down to only the code that throws the exception that you are looking for. Also without error logging it won't know if we get this error or any other error in the block.